data-raw/adlb_grading/ncictcaev5_cv.json

```diff
@@ -0,0 +1,785 @@
+[
+  {
+    "SOC": "Blood and lymphatic system disorders",
+    "TERM": "Anemia",
+    "Grade_1": "Hemoglobin (Hgb) <LLN - 10.0 g/dL; <LLN - 6.2 mmol/L; <LLN - 100 g/L",
```

Comment:
Should we unify the case, e.g., use upper case for all variables?

Reply:
Yes I can do this.